src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_FileIO.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+using Windows.ApplicationModel.Resources.Core;
+
+// Testing Append, Write and Read in one test method
+
+namespace Uno.UI.RuntimeTests.Tests
+{
+	[TestClass]
+	public class Given_FileIO
+	{
+
+		[TestInitialize]
+		public void Init()
+		{
+		}
+
+		[TestCleanup]
+		public void Cleanup()
+		{
+		}
+
+		[TestMethod]
+		public async void When_WriteReadAppendRead()

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Return 'Task' instead.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4827590)

Reply:
Codacy is totally right here: using `void` will prevent the test engine from capturing any error in the test.

Reply:
So, #2407 and #2449 have invalid tests? :(
Can you add `Task` as return value from TestMethod in these two tests, or should I create PRs for this?